hack/create-release-tag.py

```
@@ -0,0 +1,412 @@
+                    push_date = repo.get("push_date", "")
+                    return push_date[:10] if push_date else ""
+    return ""
+
+
+# ---------------------------------------------------------------------------
+# Git helpers
+# ---------------------------------------------------------------------------
+
+def git(*args, env=None) -> str:
+    """Run a git command and return stdout, raising on failure."""
+    result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)